terraform/github-to-gitlab-mirror/get-last-commit.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 github_repo"
+  exit 1
+fi
+github_repo=$1
+last_commit=$(git ls-remote "https://github.com/$github_repo" HEAD | awk '{ print $1}')
```

Comment:
Not sure how we could catch new commits that aren't on the default branch?

How big a job would it be if we dropped this script and just always pushed the repos?